src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt

```diff
@@ -364,6 +365,100 @@ class PrisonerIepLevelReviewServiceTest {
     }
   }
 
+  @Nested
+  inner class HandleSyncPatchIepReviewRequest {
+
+    // TODO: Tests to update
```

Comment:
Think you can remove this `TODO`?

Reply:
Oh yes left over I forgot, I noticed while I was checking the flaky tests again, [removed here](https://github.com/ministryofjustice/hmpps-incentives-api/pull/174/commits/4ef476e28e202fdf33549bfeb359d94e7a7f82ad) ☺️